### ISSUE TITLE
Opus files are imported as 16-bit integer

### ISSUE THREAD

**Issue by siebenbrodt** [audacity/audacity#6163]
### Bug description

Like most lossy formats, opus stores non integer values. But FFmpeg converts them to 16-bit PCM by default.

### Steps to reproduce

1. Export clipped audio to opus.
2. Import the resulting opus file.
3. Observe how all peaks above 0 dB are cut off.
4. Decode the same opus file using _ffmpeg_ with the ``-acodec pcm_f32le`` parameter (or _opusdec_ with the ``--float`` parameter).
5. Open the resulting wav and you'll see that values exceeding 0 dB remain in place.

### Expected behavior

Opus files should open with the full range they store as 32-bit float (like mp3s which are opened by Audacity with peaks above 0 dB without problems).
In addition to obviously improving accuracy, this will avoid clipping when reconverting, and allow you to see exactly how much intersample and quantization peaks are added when converting to opus.
Plus it would be consistent with the workflow, given that Audacity shows and processes imported audio as 32-bit float.

### Audacity Version

latest stable version (from audacityteam.org/download)

### Operating system

Windows

### Additional context

Audacity 3.4.2
FFmpeg Library Version: F(59.16.100),C(59.18.100),U(57.17.100)

**Comment by Cygon** [comment]
Just a bit of feedback because I stumbled into this issue while looking to file my own:

- Audacity on the 'master' branch uses `op_read_float()`
  https://github.com/audacity/audacity/blob/36918d5655ac2a87a94e0645889313835a632e2a/modules/import-export/mod-opus/ImportOpus.cpp#L202

- If I intentionally export a heavily clipped `.opus` file (3 dB above ceiling) from Audacity and load it again, the clipping is all present and accounted for.

Tested with Audacity 3,6.1 on a Gentoo system with ffmpeg 6.1.1 installed. `.opus` file opened via drag and drop.
